files/junk

```diff
@@ -0,0 +1,7 @@
+# This is just to keep notes on snmp commands to run to test stuff.
```

Comment:
If there are noted that you'd like users to know about they should be in README.md